client/fingerprint/host.go

```diff
@@ -38,11 +35,7 @@ func (f *HostFingerprint) Fingerprint(cfg *config.Config, node *structs.Node) (b
 	node.Attributes["kernel.version"] = ""
 
 	if runtime.GOOS != "windows" {
```

Comment:
KernelVersion will be an empty string on platforms without support so you can remove this guard and set `kernel.version` above directly.